src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,701 @@
+
+**Nano Stores et les [Stores de Svelte](https://svelte.dev/tutorial/writable-stores) sont très similaires!** En effet, [Nano Stores vous permettent d'utiliser le même raccourci `$`](https://github.com/nanostores/nanostores#svelte) pour les abonnements que vous pourriez utiliser avec Svelte stores.
+
+Si vous voulez éviter les bibliothèques tierces, les [Stores de Svelte](https://svelte.dev/tutorial/writable-stores) sont à eux seuls un excellent outil de communication inter-processus. Néanmoins, vous préférerez peut-être Nano Stores si :
+1. Vous aimez leurs modules complémentaires pour [les objets](https://github.com/nanostores/nanostores#maps) et [l'état asynchrone](https://github.com/nanostores/nanostores#lazy-stores).
+2. Ou si vous souhaitez communiquer entre Svelte et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
+</details>
+
+<details>
+<summary>**🙋 Comment Solid signals est-il différent de Nano Stores?**</summary>
+
+Si vous utilisez Solid depuis un certain temps, vous avez peut-être essayé de déplacer les [signals](https://www.solidjs.com/docs/latest#createsignal) ou [stores](https://www.solidjs.com/docs/latest#createstore) en dehors de vos composants. C'est un excellent moyen de partager l'état entre les Îles de composants Solid ! Essayez d'exporter des "signals" à partir d'un fichier partagé :
+
+```js
+// sharedStore.js
+import { createSignal } from 'solid-js';
+
+export const sharedCount = createSignal(0);
+```
+...et tous les composants important `sharedCount` partageront le même état. Bien que cela fonctionne bien, vous préférerez peut-être Nano Stores si a) vous aimez leurs modules complémentaires pour
+ ["objects"](https://github.com/nanostores/nanostores#maps) et [async state](https://github.com/nanostores/nanostores#lazy-stores), ou b) vous souhaitez communiquer entre Solid et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
+
+</details>
+:::
+
+## Installer Nano Stores
+
+Pour commencer, installez Nano Stores avec leur package d'assistance pour votre framework d'interface utilisateur préféré :
+
+<UIFrameworkTabs>
+  <Fragment slot="preact">
+  ```shell
+  npm install nanostores @nanostores/preact
+  ```
+  </Fragment>
+  <Fragment slot="react">
+  ```shell
+  npm install nanostores @nanostores/react
+  ```
+  </Fragment>
+  <Fragment slot="solid">
+  ```shell
+  npm install nanostores @nanostores/solid
+  ```
+  </Fragment>
+  <Fragment slot="svelte">
+  ```shell
+  npm install nanostores
+  ```
+  :::note
+  Pas de package d'assistance ici ! Nano Stores peuvent être utilisés comme Svelte stores standard
+
+  :::
+  </Fragment>
+  <Fragment slot="vue">
+  ```shell
+  npm install nanostores @nanostores/vue
+  ```
+  </Fragment>
+</UIFrameworkTabs>
+
+Vous pouvez aller directement dans le [Guide d'utilisation de Nano Stores](https://github.com/nanostores/nanostores#guide) à partir d'ici, ou suivez notre exemple ci-dessous!
```

Comment:
Just a little typo error for the end of this phrase. 

Add a space between `ci-dessous` and `!` 😁

Reply:
Woah! Just in time — I was about to merge. Could you add these as “suggestions”? That way you can get credit:

<img width="698" alt="image" src="https://user-images.githubusercontent.com/357379/202745859-98145e7d-54c4-43fa-877c-884c56d06a9f.png">

Reply:
Ohh yeas I clicked to the wrong button for the review changes 😵

Reply:
Oooh founded!